src/components/menu/menu-popover.tsx

```diff
@@ -22,14 +22,16 @@ export const calculatePopoverPosition = (
   }
 }
 
-export const MenuPopover: FC<{
-  /**
-   * Optional parameter to adjust the vertical position of the popover
-   *
-   * @default 4 // confirmed by design team to use var(--spacing-1) initially
-   */
-  yOffset?: number
-}> = ({ children, yOffset = 4 }) => {
+export const MenuPopover: FC<
+  {
+    /**
+     * Optional parameter to adjust the vertical position of the popover
+     *
+     * @default 4 // confirmed by design team to use var(--spacing-1) initially
+     */
+    yOffset?: number
+  } & HTMLAttributes<HTMLDivElement>
+> = ({ children, yOffset = 4, ...props }) => {
```

Comment:
**question:** What's the need for the `...props` here?

Reply:
to allow the `TopBar` react consumer to use defined yOffset style like this

![image](https://github.com/user-attachments/assets/6a15cd20-e8fa-48b4-aa6f-9135372c1661)
 
but essentially we just need the classname though, should I update it? 🤔

Reply:
Ahh, of course, sorry. I saw that addition, but didn't connect it to this change here. This is fine 👍